### ISSUE TITLE
Remove OI file manager from YouTube videos

### ISSUE THREAD

**Issue by elmps2018** [opendatakit/tool-suite-X#299]
On our YouTube [channel](https://www.youtube.com/channel/UCxOZGaf3xUiD85q8qJaw2bQ), there are a number of videos demo-ing various features of ODK-X. Some probably include OI file manager, which is going away as a requirement in v2.1.9. Using the preview [release](https://forum.odk-x.org/t/preview-release-odk-x-tool-suite-v2-1-9/1527) for v2.1.9 please redo videos that had OI file manager to show instead operations using Google Files. 

If there is more than one video that needs revising, more than one person can work on this issue -- please just review videos, comment which video you find that needs revising and we can split the issue.

**Comment by chitranshix** [comment]
I'm up for it. Please assign me this issue.

**Comment by elmps2018** [comment]
@apigeoneer assigned to you! Let us know any questions! If you can let us know what videos you are working on and share revised videos on google drive as they are ready that would be great.

**Comment by chitranshix** [comment]
Thanks! I can start with the video [How to install basic ODK-X tools on your tablet](https://www.youtube.com/watch?v=_387vP9CO9M). It includes OI file manager installation.

Do I need to recreate the whole video & share my version or download the YouTube video, edit it so as to remove the parts concerning the OI file manager & share that?

**Comment by EstherPhebe** [comment]
@apigeoneer You can choose to either recreate the video or download and edit the current video, just decide what works best for you and share the videos for feedback.

**Comment by chitranshix** [comment]
Sure. Thanks.

**Comment by chitranshix** [comment]
Here is a [Google Drive link](https://drive.google.com/file/d/1bqHu5RY0UHBWsHuPcHIR0tOrYHYrbKKT/view?usp=sharing) of the edited video. Let me know if any more changes are needed.
Now I'm working on the [Instalar apps ODK-X (Install apps in ODK-X: Spanish)](https://www.youtube.com/watch?v=jrGSySKK9p8) video.

**Comment by elmps2018** [comment]
@apigeoneer thanks for working on this! So it looks like this points out that the file manager, shown in downloads, is not needed in v2.1.9. @EstherPhebe @wbrunette or @jbeorse do you think it is helpful to mention file manager even to say it is not needed or confusing? Not sure which myself, but we should pick an approach to be consistent across videos.

**Comment by wbrunette** [comment]
It is not needed to install and use. It's only needed to clear your device currently, which is very useful to know how to do. However it's no longer a prerequisite to have it installed.

**Comment by elmps2018** [comment]
@wbrunette, right, but the question is do we want that it is NOT needed in 2.1.9 mentioned in the video, or not at all because it adds to confusion and just show with files by google instead?

**Comment by wbrunette** [comment]
It should probably only be mentioned in the clearing the setup video, which needs to be redone. https://www.youtube.com/watch?v=KNogiu-crzA

**Comment by wbrunette** [comment]
To clarify we should remove OI file manager completely from the videos

**Comment by chitranshix** [comment]
Here are the newly edited videos -

I've removed any mention of OI file manager completely from these -
[How to install basic ODK-X tools on your tablet](https://drive.google.com/file/d/1G75qnf-gxMM628riNDzlYUlOtnQuQYut/view?usp=sharing) (This video has OI file manager in the background. Is that okay?)
[Instalar apps ODK-X (Install apps in ODK-X: Spanish)](https://drive.google.com/file/d/1ee0gFgWFfqe-lNQPimyJf58f_uWaMxSC/view?usp=sharing) (But this one uses v 2.1.7 & calls it the latest version. I think it needs to be recreated, though it isn't mentioned in this issue. I don't know Spanish.)

As suggested by @wbrunette, I've mentioned that OI file manager isn't required in these. I can remove it completely too. Let me know.
[How To Delete All ODK-X Data](https://drive.google.com/file/d/1U7dNqk-zZA5uu6vGaVCxjdtm1cLnkG3q/view?usp=sharing)
[Borrar Datos ODK-X (How to delete all the data in ODK-X: Spanish)](https://drive.google.com/file/d/1PmmNtYpar5csvZsJDPiP1ouLjQV0VkwA/view?usp=sharing)

These use IO file manager as their file manager but they don't mention that it is required by ODK-X. Removing OI file manager from them would mean creating a new video using some other file manager. Should I do that? Or should I mention in the old videos that any file manager can be used?
[How to Import Data to ODK-X Tables](https://www.youtube.com/watch?v=Kzut6Xrv3SM)
[Importar fichero csv (Importing a csv into ODK-X tables: Spanish)](https://www.youtube.com/watch?v=9NfXSBkU6gI)
[How to Export Data in ODK-X Tables](https://www.youtube.com/watch?v=zqp69HA77eo)
[Exportar Datos (Exporting data from ODK-X tables: Spanish)](https://www.youtube.com/watch?v=S4ACjU-lqY8)
[How To Zip And Share ODK-X Data](https://www.youtube.com/watch?v=jzqu7w4VepM)
[Comprimir y Enviar (How to zip and share ODK-X data: Spanish)](https://www.youtube.com/watch?v=2cs_Ec48OmM)

**Comment by elmps2018** [comment]
Hi @apigeoneer thanks for working on these! @wbrunette and @apigeoneer what do you recommend? It seemed like @wbrunette you were suggesting recreating the videos, which I think makes sense to show with Files by Google instead.

**Comment by wbrunette** [comment]
I think you want to recreate the How to Delete All ODK-X Data using files by Google... that is probably the most important one

**Comment by chitranshix** [comment]
Okay. I'm recreating the ones in English.

**Comment by chitranshix** [comment]
Here are the recreated videos. Please have a look & suggest any changes.

[How to Import Data in ODK-X Tables
](https://drive.google.com/file/d/1KVEtUPaC7K3E3UtnqCCw1evQ7KixeFmH/view?usp=sharing)
[How to Export Data from ODK-X Tables
](https://drive.google.com/file/d/1x24zMCna-xHpxmt8_5p5KMXmdDG3u8Hb/view?usp=sharing)
[How to Delete All ODK-X Data
](https://drive.google.com/file/d/1a48aU-8rz8ixin1aLq0tuFPKL-hRTvkj/view?usp=sharing)
[How to Install Basic ODK-X Tools](https://drive.google.com/file/d/17IHo1Tpyrro-VpY6oOE0lOvev23ElLRO/view?usp=sharing)
How to Zip & Share ODK-X Data

**Comment by wbrunette** [comment]
@elmps2018 and @EstherPhebe note the new videos. 

@apigeoneer a link is not showing for the Zip and Share data

**Comment by elmps2018** [comment]
@apigeoneer thanks for working on these! I am reviewing my way through and commenting as I go to help keep track. The ODK-X Tables importing one looks good to me, @EstherPhebe did spot any issues or does it look good to you too? I am adding this to the 2.1.9 milestone since we would want to put these out with 2.1.9 version.

**Comment by elmps2018** [comment]
How to export looks good too.

**Comment by elmps2018** [comment]
How to delete looks good to me.

**Comment by elmps2018** [comment]
For the "How to install" @wbrunette do we want to say any app at all or recommend Files by Google?

**Comment by apigeoneer** [comment]
Hi! Files by Google doesn't have an option to compress into a zip folder, so I used my default file manager for that video. Should I share that one or make another by using Files & some online compressor?

**Comment by apigeoneer** [comment]
Also, for some reason, the APKs weren't getting installed on my phone using Files by Google, so I used the default file manager in that one.

> @elmps2018 and @EstherPhebe note the new videos.
> 
> @apigeoneer a link is not showing for the Zip and Share data

Hi! Files by Google doesn't have an option to compress into a zip folder, so I used my default file manager for that video. Should I share that one or make another by using Files & some online compressor?

**Comment by chitranshix** [comment]
> @elmps2018 and @EstherPhebe note the new videos.
> 
> @apigeoneer a link is not showing for the Zip and Share data

Hi! Files by Google doesn't have an option to compress into a zip folder, so I used my default file manager for that video. Should I share that one or make another by using Files & some online compressor?

**Comment by wbrunette** [comment]
> do we want to say any app at all or recommend Files by Google?

I think we want to say we recommend Files by Google.

**Comment by elmps2018** [comment]
@apigeoneer I think for install we want to recommend Files by Google and show that one. I think it's less important for the zip.

**Comment by chitranshix** [comment]
Hi! I've been trying for quite some time now, but for some reason,  the installation of all APKs (both v2.1.8 & the preview version) using Files by Google is failing with the snack bar "Something went wrong. The app was not installed". Can anyone suggest why this might be happening?
I'm using Redme Note 8 Pro.

**Comment by elmps2018** [comment]
@apigeoneer thanks for persisting! Can you ask on the Slack if anyone else has tried with the Files by Google and what happened? And with what tablet/phone version? Then we can narrow down what the issue is a bit.

**Comment by chitranshix** [comment]
Sure.

**Comment by wbrunette** [comment]
Files by Google you should be able to install from the Google Play store.

You should use Chrome to install ODK-X Services, Survey, and Tables

**Comment by wbrunette** [comment]
Also, Chrome has to be authorize to allow the installation of APKs as shown in the current video

**Comment by wbrunette** [comment]
@elmps2018 is this complete? If a part is not complete from this discussion maybe we can make an issue for it?

**Comment by elmps2018** [comment]
@wbrunette this is not complete. Isn't it an issue currently?
